hack/release.sh

```diff
@@ -13,5 +13,5 @@ echo "Building container image"
 
 # Using defaults, this pushes to:
 # quay.io/shipwright/shipwright-build-controller:latest
-KO_DOCKER_REPO="$IMAGE_HOST/$IMAGE" GOFLAGS="${GO_FLAGS}" ko resolve -t "$TAG" --bare -R -f deploy/ > release.yaml
-KO_DOCKER_REPO="$IMAGE_HOST/$IMAGE" GOFLAGS="${GO_FLAGS} -tags=pprof_enabled" ko resolve -t "$TAG-debug" --bare -R -f deploy/ > release-debug.yaml
+KO_DOCKER_REPO="$IMAGE_HOST/$IMAGE" GOFLAGS="${GO_FLAGS}" ko resolve -t "$TAG" --bare --platform=all -R -f deploy/ > release.yaml
+KO_DOCKER_REPO="$IMAGE_HOST/$IMAGE" GOFLAGS="${GO_FLAGS} -tags=pprof_enabled" ko resolve -t "$TAG-debug" --bare --platform=all -R -f deploy/ > release-debug.yaml
```

Comment:
For reference - `ko` inspects the manifest list of the base image, and then does a golang build for each platform in the manifest [1]. I assume this also means `ko resolve` will push a manifest list.

[1] https://github.com/google/ko#multi-platform-images

Reply:
Yep, that's how it looks: https://hub.docker.com/r/saschaschwarze/shipwright-build-controller/tags?page=1&ordering=last_updated